model/task_queue_service_dependency.go

```diff
@@ -574,3 +550,27 @@ func (d *basicCachedDAGDispatcherImpl) nextTaskGroupTask(unit schedulableUnit) *
 
 	return nil
 }
+
+func checkUnmarkedBlockingTasks(t *task.Task, dependencyCaches map[string]task.Task) error {
+	catcher := grip.NewBasicCatcher()
+
+	blockingTasks, err := t.RefreshBlockedDependencies(dependencyCaches)
+	catcher.Add(errors.Wrap(err, "can't get blocking tasks"))
+	if err == nil && len(blockingTasks) > 0 {
```

Comment:
You don't actually need to check len(blockingTasks) > 0, the for loop just won't do anything in that case

Reply:
removed